lms/djangoapps/ccx/views.py

```diff
@@ -140,6 +140,11 @@ def dashboard(request, course, ccx=None):
         schedule = get_ccx_schedule(course, ccx)
         grading_policy = get_override_for_ccx(
             ccx, course, 'grading_policy', course.grading_policy)
+        # TODO: this logic will change once child course bug is fixed.
+        # This value will be later retrieved from ccx override.
+        context['course_display_name'] = ccx.display_name
```

Comment:
@taranjeet Two notes here:

* I might be missing something, but it doesn't look like we'll need to make any changes to this code once #18808 merges: At this point in the control flow we have access to the `CustomCourseForEdX` object that represents the CCX course that we want to load the CCX Coach tab for (it is referenced by the `ccx` variable). So it's perfectly fine to do `ccx.display_name` to retrieve the name of the CCX course.
* `context` already contains a reference to `ccx` (cf. L132 above), so you could just do `ccx.display_name` in the template and remove this line.

Reply:
Ohh! I see. We can use course name from `ccx.display_name` and hence there won't be any need to pass this as a variable in context.
This means that this part, in particular, does not depend on "Child Course Bug fix PR" #18808